src/views/ContinuationIn/ContinuationInBusinessHome.vue

```diff
@@ -2,7 +2,7 @@
   <div id="continuation-in-business-home">
     <!-- Existing Business Information -->
     <section class="mt-10">
-      <header id="existing-Business-information">
+      <header id="existing-business-information">
```

Comment:
A 1-character bug.

When this section was invalid, the page didn't scroll to it. Turns out, ids are case-sensitive 🤦